packages/frontend/src/retro/components/dialogs/CreateTimerDialog.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button, Dialog, DialogContent, DialogTitle, DialogActions } from "@mui/material";
+import { CallToActionButton } from "../../../common/components/buttons/CallToActionButton";
+import { DialogProps } from "../../../common/types/commonTypes";
+import { useFullscreen } from "../../hooks/useFullscreen";
+import { TimePicker } from "../../../common/components/TimePicker";
+import React = require("react");
+import { useValidatedTimeInput } from "../../../common/hooks/useValidatedTimeInput";
+
+interface CreateTimerDialogProps extends DialogProps {
+  startTimer: (duration: number) => void;
+  stopTimer: () => void;
+  pauseTimer: () => void;
+  isTimerRunning: boolean;
+  isTimerPaused: boolean;
+  remainingMinutes: number;
+  remainingSeconds: number;
+}
+export function CreateTimerDialog({
+  isOpen,
+  close,
+  startTimer,
+  stopTimer,
+  pauseTimer,
+  isTimerRunning,
+  isTimerPaused,
+  remainingMinutes,
+  remainingSeconds,
+}: CreateTimerDialogProps) {
+  const fullScreen = useFullscreen();
+
+  const {
+    value: pickedSeconds,
+    formattedValue: formattedSeconds,
+    isError: isSecondsError,
+    onChange: onSecondsChange,
+  } = useValidatedTimeInput({
+    formatLength: 2,
+    maxValue: 60,
+    minValue: 0,
+    initialValue: remainingSeconds,
+  });
+
+  const {
+    value: pickedMinutes,
+    formattedValue: formattedMinutes,
+    isError: isMinutesError,
+    onChange: onMinutesChange,
+    incrementTime: onMinutesIncrement,
+    decrementTime: onMinutesDecrement,
```

Comment:
Rename to handleMinutesIncrement and handleMinutesDecrement

these are functions that do something. We use "on" when a function provides a way to set something, e.g. in a function parameter